### PR TITLE
[MP-18] Slider component v2

### PR DESCRIPTION
**Pull request by thisisofficial**
### Description

Added an image slider with customizable image backgrounds and titles.
Vídeo sin título ‐ Hecho con Clipchamp

### Testing Checklist

- [ ] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [MP-18](https://cd95.atlassian.net/browse/MP-18)

### CODE REVIEW

**Comment by thisisofficial** [comment]
@caarlosdamian Review please

**Comment by thisisofficial** [comment]
Review again please, this has been long overdue

File: src/components/imageCarousel/ImageSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      {dots && (
+        <div className="container-dots">
+          {data.map((slide, index) => (
+            <button
+              type="button"
+              key={`dot${slide.title}`}
+              onClick={() => moveDot(index)}
+              className={slideIndex === index ? 'dot active' : 'dot'}
+            >
+              {' '}
```
Comment (caarlosdamian) [inline src/components/imageCarousel/ImageSlider.tsx:88]: remove this

File: src/components/imageCarousel/imageSlider.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  margin: 100px auto 0;
+  position: relative;
+  overflow: hidden;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.2);
+  
+}
+
+
+.slider-container {
+  /* transform: translate(-800px); */
```
Comment (caarlosdamian) [inline src/components/imageCarousel/imageSlider.css:13]: remove this

File: src/components/imageCarousel/imageSlider.css [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  height: 100%;
+}
+
+.img-container img{
+  object-fit: cover;
+  height: 100%;
+  box-shadow: 0 -100px 200px rgba(50, 50, 50, 0.9) inset;
+}
+
+.circle{
```
Comment (caarlosdamian) [inline src/components/imageCarousel/imageSlider.css:129]: why do you have this class????

File: src/components/imageCarousel/imageSlider.css [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  height: 50px;
+  border-radius: 50%;
+  background: red;
+  position: absolute;
+  top: 50%;
+  left: 30%;
+  cursor: pointer;
+}
+
+.circle2{
```
Comment (caarlosdamian) [inline src/components/imageCarousel/imageSlider.css:140]: why do you have this class???? remove if you dont use it

**Review by caarlosdamian — CHANGES_REQUESTED** [review]
check

**Review by caarlosdamian — APPROVED** [review]

**Pull request merged**